Intermittent failure in `wallet_groups.py`

https://cirrus-ci.com/task/6569439897321472?logs=functional_tests#L2462 : 

```
File "C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\test\functional\wallet_groups.py", line 150, in run_test
                                       txid6 = self.nodes[4].sendtoaddress(self.nodes[0].getnewaddress(), 2.95)
(...)
AssertionError: [node 4] Expected messages "['Fee non-grouped = 5520, grouped = 8240, using grouped']" does not partially match log

After running it several times on my machine, I got same error:
```sh
2022-08-29T13:59:03.850000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_framework.py", line 133, in main
    self.run_test()
  File "/Users/brunogarcia/projects/bitcoin-core-dev/./test/functional/wallet_groups.py", line 151, in run_test
    txid6 = self.nodes[4].sendtoaddress(self.nodes[0].getnewaddress(), 2.95)
  File "/opt/homebrew/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/contextlib.py", line 126, in __exit__
    next(self.gen)
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_node.py", line 423, in assert_debug_log
    self._raise_assertion_error('Expected messages "{}" does not partially match log:\n\n{}\n\n'.format(str(expected_msgs), print_log))
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_node.py", line 167, in _raise_assertion_error
    raise AssertionError(self._node_msg(msg))
AssertionError: [node 4] Expected messages "['Fee non-grouped = 5520, grouped = 8240, using grouped']" does not partially match log:

 - 2022-08-29T13:59:01.964574Z [http] [httpserver.cpp:241] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:49661
 - 2022-08-29T13:59:01.964783Z [httpworker.1] [rpc/request.cpp:179] [parse] [rpc] ThreadRPCServer method=sendtoaddress user=__cookie__
 - 2022-08-29T13:59:01.967513Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool added 1 keys (0 internal), size=2 (1 internal)
 - 2022-08-29T13:59:02.020424Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool reserve 2
 - 2022-08-29T13:59:02.020990Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021075Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021147Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021224Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021306Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021404Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.023758Z [httpworker.1] [wallet/coinselection.cpp:320] [KnapsackSolver] [selectcoins] Coin selection best subset: 1.00 1.00 1.00 total 2.99995917
 - 2022-08-29T13:59:02.081274Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool keep 2
 - 2022-08-29T13:59:02.081476Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee Calculation: Fee:5524 Bytes:276 Tgt:2 (requested 6) Reason:"Half Target 60% Threshold" Decay 0.96200: Estimation: (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.083902Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool added 1 keys (1 internal), size=2 (1 internal)
 - 2022-08-29T13:59:02.137461Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool reserve 4
 - 2022-08-29T13:59:02.137973Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138055Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138124Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138203Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138272Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138350Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.200202Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool keep 4
 - 2022-08-29T13:59:02.200361Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee Calculation: Fee:8245 Bytes:412 Tgt:2 (requested 6) Reason:"Half Target 60% Threshold" Decay 0.96200: Estimation: (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.200418Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee non-grouped = 5524, grouped = 8245, using non-grouped
 - 2022-08-29T13:59:02.200556Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] CommitTransaction:
 - CTransaction(hash=63de543f9f, ver=2, vin.size=3, vout.size=2, nLockTime=88)
 -     CTxIn(COutPoint(bd00ef514b, 0), scriptSig=, nSequence=4294967293)
 -     CTxIn(COutPoint(97c1451f61, 1), scriptSig=, nSequence=4294967293)
 -     CTxIn(COutPoint(782bc30dbd, 0), scriptSig=, nSequence=4294967293)
 -     CScriptWitness(304402206e5bec091b1f13878b5a728312d2b5f149300872dfaa94b94da347ce1cf2de7102203aae2651dcdbf7f32642884521eafcc25db5227714d9a8878639ecec0993ecd801, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CScriptWitness(304402203bd40ae7040bb065acb3fa1b1312dc6c647144db3c98c7dccd01d34be92f0aca02201d2c37ff8e9eeec6c96515529f92e401b9df5ce30aca598b09b3b701fcc99ea901, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CScriptWitness(304402204822d244b3b704a670446ea0cb2faa27300d33352ca72a3b3f9d85fe659090090220575f8e8785031e403d10ff0771b4c4baa178f52b77a6205ed8aa7719ce22184e01, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CTxOut(nValue=0.04994476, scriptPubKey=00142217fba66bb0e12889e672d587)
 -     CTxOut(nValue=2.95000000, scriptPubKey=0014debb3dd9d593090d70ce9e1138)
 - 2022-08-29T13:59:02.200694Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] AddToWallet 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6  newupdate
 - 2022-08-29T13:59:02.254239Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Submitting wtx 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 to mempool for relay
 - 2022-08-29T13:59:02.255631Z [httpworker.1] [txmempool.cpp:736] [check] [mempool] Checking mempool with 1 transactions and 3 inputs
 - 2022-08-29T13:59:02.255835Z [httpworker.1] [validationinterface.cpp:211] [TransactionAddedToMempool] [validation] Enqueuing TransactionAddedToMempool: txid=63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 wtxid=019e676a04529e0d790c1716fd8b803570967d13477fc83ed6832ce78e868951
 - 2022-08-29T13:59:02.255910Z [httpworker.1] [txmempool.cpp:736] [check] [mempool] Checking mempool with 2 transactions and 6 inputs
 - 2022-08-29T13:59:02.255985Z [scheduler] [validationinterface.cpp:211] [operator()] [validation] TransactionAddedToMempool: txid=63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 wtxid=019e676a04529e0d790c1716fd8b803570967d13477fc83ed6832ce78e868951
 - 2022-08-29T13:59:02.256412Z [scheduler] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] AddToWallet 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6  2022-08-29T13:59:03.850000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_framework.py", line 133, in main
    self.run_test()
  File "/Users/brunogarcia/projects/bitcoin-core-dev/./test/functional/wallet_groups.py", line 151, in run_test
    txid6 = self.nodes[4].sendtoaddress(self.nodes[0].getnewaddress(), 2.95)
  File "/opt/homebrew/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/contextlib.py", line 126, in __exit__
    next(self.gen)
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_node.py", line 423, in assert_debug_log
    self._raise_assertion_error('Expected messages "{}" does not partially match log:\n\n{}\n\n'.format(str(expected_msgs), print_log))
  File "/Users/brunogarcia/projects/bitcoin-core-dev/test/functional/test_framework/test_node.py", line 167, in _raise_assertion_error
    raise AssertionError(self._node_msg(msg))
AssertionError: [node 4] Expected messages "['Fee non-grouped = 5520, grouped = 8240, using grouped']" does not partially match log:

 - 2022-08-29T13:59:01.964574Z [http] [httpserver.cpp:241] [http_request_cb] [http] Received a POST request for / from 127.0.0.1:49661
 - 2022-08-29T13:59:01.964783Z [httpworker.1] [rpc/request.cpp:179] [parse] [rpc] ThreadRPCServer method=sendtoaddress user=__cookie__
 - 2022-08-29T13:59:01.967513Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool added 1 keys (0 internal), size=2 (1 internal)
 - 2022-08-29T13:59:02.020424Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool reserve 2
 - 2022-08-29T13:59:02.020990Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021075Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021147Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021224Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021306Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.021404Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.023758Z [httpworker.1] [wallet/coinselection.cpp:320] [KnapsackSolver] [selectcoins] Coin selection best subset: 1.00 1.00 1.00 total 2.99995917
 - 2022-08-29T13:59:02.081274Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool keep 2
 - 2022-08-29T13:59:02.081476Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee Calculation: Fee:5524 Bytes:276 Tgt:2 (requested 6) Reason:"Half Target 60% Threshold" Decay 0.96200: Estimation: (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.083902Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool added 1 keys (1 internal), size=2 (1 internal)
 - 2022-08-29T13:59:02.137461Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool reserve 4
 - 2022-08-29T13:59:02.137973Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138055Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138124Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138203Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 1 > 60% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138272Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 2 > 85% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.138350Z [httpworker.1] [policy/fees.cpp:381] [EstimateMedianVal] [estimatefee] FeeEst: 4 > 95% decay 0.96200: feerate: 20010.6 from (19613.1 - 1e+99) 100.00% 13.3/(13.3 0 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.200202Z [httpworker.1] [wallet/scriptpubkeyman.h:248] [WalletLogPrintf] [default wallet] keypool keep 4
 - 2022-08-29T13:59:02.200361Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee Calculation: Fee:8245 Bytes:412 Tgt:2 (requested 6) Reason:"Half Target 60% Threshold" Decay 0.96200: Estimation: (19613.1 - 1e+99) 86.05% 12.3/(13.3 1 mem 0.0 out) Fail: (0 - 19613.1) 0.00% 0.0/(0.0 0 mem 0.0 out)
 - 2022-08-29T13:59:02.200418Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Fee non-grouped = 5524, grouped = 8245, using non-grouped
 - 2022-08-29T13:59:02.200556Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] CommitTransaction:
 - CTransaction(hash=63de543f9f, ver=2, vin.size=3, vout.size=2, nLockTime=88)
 -     CTxIn(COutPoint(bd00ef514b, 0), scriptSig=, nSequence=4294967293)
 -     CTxIn(COutPoint(97c1451f61, 1), scriptSig=, nSequence=4294967293)
 -     CTxIn(COutPoint(782bc30dbd, 0), scriptSig=, nSequence=4294967293)
 -     CScriptWitness(304402206e5bec091b1f13878b5a728312d2b5f149300872dfaa94b94da347ce1cf2de7102203aae2651dcdbf7f32642884521eafcc25db5227714d9a8878639ecec0993ecd801, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CScriptWitness(304402203bd40ae7040bb065acb3fa1b1312dc6c647144db3c98c7dccd01d34be92f0aca02201d2c37ff8e9eeec6c96515529f92e401b9df5ce30aca598b09b3b701fcc99ea901, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CScriptWitness(304402204822d244b3b704a670446ea0cb2faa27300d33352ca72a3b3f9d85fe659090090220575f8e8785031e403d10ff0771b4c4baa178f52b77a6205ed8aa7719ce22184e01, 03f2660628e57b7ec921650248b41e1b270ff84332634776d79f1215fffa1585a0)
 -     CTxOut(nValue=0.04994476, scriptPubKey=00142217fba66bb0e12889e672d587)
 -     CTxOut(nValue=2.95000000, scriptPubKey=0014debb3dd9d593090d70ce9e1138)
 - 2022-08-29T13:59:02.200694Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] AddToWallet 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6  newupdate
 - 2022-08-29T13:59:02.254239Z [httpworker.1] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] Submitting wtx 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 to mempool for relay
 - 2022-08-29T13:59:02.255631Z [httpworker.1] [txmempool.cpp:736] [check] [mempool] Checking mempool with 1 transactions and 3 inputs
 - 2022-08-29T13:59:02.255835Z [httpworker.1] [validationinterface.cpp:211] [TransactionAddedToMempool] [validation] Enqueuing TransactionAddedToMempool: txid=63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 wtxid=019e676a04529e0d790c1716fd8b803570967d13477fc83ed6832ce78e868951
 - 2022-08-29T13:59:02.255910Z [httpworker.1] [txmempool.cpp:736] [check] [mempool] Checking mempool with 2 transactions and 6 inputs
 - 2022-08-29T13:59:02.255985Z [scheduler] [validationinterface.cpp:211] [operator()] [validation] TransactionAddedToMempool: txid=63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6 wtxid=019e676a04529e0d790c1716fd8b803570967d13477fc83ed6832ce78e868951
 - 2022-08-29T13:59:02.256412Z [scheduler] [wallet/wallet.h:827] [WalletLogPrintf] [default wallet] AddToWallet 63de543f9faf012dc93b11758723c6e54b22c5c84e1683e2c36dcfd7bc6934c6  
```

--- a/test/functional/wallet_groups.py
+++ b/test/functional/wallet_groups.py
@@ -26,9 +26,10 @@ class WalletGroupTest(BitcoinTestFramework):
             ["-maxapsfee=0.00002719"],
             ["-maxapsfee=0.00002720"],
         ]
-        # whitelist peers to speed up tx relay / mempool sync
+
         for args in self.extra_args:
-            args.append("-whitelist=noban@127.0.0.1")
+            args.append("-whitelist=noban@127.0.0.1")   # whitelist peers to speed up tx relay / mempool sync
+            args.append(f"-paytxfee={20 * 1e3 / 1e8}")  # apply feerate of 20 sats/vB across all nodes
 
         self.rpc_timeout = 480
 

